akka-actor/src/main/scala/akka/actor/dungeon/FaultHandling.scala

```diff
@@ -244,7 +244,7 @@ private[akka] trait FaultHandling { this: ActorCell =>
       if (freshActor eq failedActor) setActorFields(freshActor, this, self) // If the creator returns the same instance, we need to restore our nulled out fields.
 
       freshActor.aroundPostRestart(cause)
-      checkReceiveTimeout() // user may have set a receive timeout in preStart which is called from postRestart
+      checkReceiveTimeout(reschedule = true) // user may have set a receive timeout in preStart which is called from postRestart
```

Comment:
is it only ever called with `reschedule = true`? then we don't need the parameter at all?

Reply:
The second use is internal to `ReceiveTimeout` which does leverage the boolean:
https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/actor/dungeon/ReceiveTimeout.scala#L32

#28385 ads the same as `ActorCell` to `FaultHandler`, passing in `true`.

Reply:
good, otherwise I would be worried we missed something in previous refactorings here